.vscode/tasks.json

```
@@ -0,0 +1,62 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
+  // for the documentation about the tasks.json format
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Build (Jekyll)",
+      "type": "shell",
+      "command": "bundle",
+      "args": [
+        "exec",
+        "jekyll",
+        "build"
+      ],
```

Missing `JEKYLL_ENV`:

```suggestion
      ],
      "options": {
        "env": {
          "JEKYLL_ENV": "production"
        }
      },
```